webvtt/rendering/cues-with-video/processing-model/basic.html

```diff
@@ -2,6 +2,7 @@
 <html class="reftest-wait">
```

Comment:
Wait, does `class="reftest-wait"` and including `/fonts/ahem.css` play well together?

From https://github.com/web-platform-tests/wpt/blob/master/tools/wptrunner/wptrunner/executors/reftest-wait_webdriver.js it looks like when using `class="reftest-wait"` one would have to also wait for `document.fonts.ready` to not change behavior.

@Hexcles has also had fun with reftest timing and might be able to provide quick background on how we arrived at the current state of affairs.

Reply:
Because of the `if (!root.classList.contains("reftest-wait"))` condition.

Reply:
Sure, which just means we don't take a screenshot until the first time `root_wait` is called after `reftest-wait` is removed?

Reply:
Like, there's no path from that condition being false to the callback being called, is there?

Reply:
It's hard to fiddle with this since only the webvtt tests are using `reftest-wait` (in this PR) and they're already failing (as per the other comment). But there are certainly other tests in #17173 that use `reftest-wait` and aren't affected, so I'm leaning towards assuming that `reftest-wait` and `/fonts/ahem.css` do play well together.

Reply:
Aha, so I missed the `new MutationObserver(root_wait)`, it all makes sense now :)